website/docs/tutorials/07-testing/03-fuzzing/strategies.md

```diff
@@ -0,0 +1,62 @@
+# Strategies
+
+While a felt is a _lingua franca_ of Cairo runtime, its whole value space is rarely covered by
+domain space in tested code.
+
+Using the [`given`](../02-cheatcodes/given.md) cheatcode,
+one can instruct Protostar how constrained or adjusted the value space of fuzzed inputs should be.
+
+By default, Protostar applies the [`strategy.unsigned()`](#strategyunsigned) strategy to all felt
+parameters.
```

Comment:
Line breaks could be removed.

![Screen Shot 2022-08-04 at 17 59 08](https://user-images.githubusercontent.com/9629577/182893680-43ebe363-b4a8-42b6-9cc8-7b6911c1d5ff.png)

Reply:
This section doesn't answer the question: "What is a Strategy?" (and how is a strategy related to the `given` cheatcode).

Reply:
I don't agree that line breaks could be removed. These are separate paragraphs that say independent things.

Reply:
Done.